### ISSUE TITLE
Eliminate division warning of blackbody_wn_rad2temp

### ISSUE THREAD

**Issue by chorng** [pytroll/pyspectral#238]
#### Code Sample, a minimal, complete, and verifiable piece of code
```python
with np.errstate(divide='ignore', invalid='ignore'):
    return PLANCK_C1 * wavenumber / np.log((PLANCK_C2 * wavenumber**3) / radiance + 1.0)

```

#### Problem description
The blackbody_wn_rad2temp function is currently raising runtime warning when encountering divide by zero, which is not the same behavior as other functions, e.g. the planck function.

#### Expected Output
No warning is raised when encountering divide by zero.

#### Actual Result, Traceback if applicable
RuntimeWarning: divide by zero encountered in divide
  return PLANCK_C1 * wavenumber / np.log((PLANCK_C2 * wavenumber**3) / radiance + 1.0)

#### Versions of Python, package at hand and relevant dependencies
Python 3.10, pyspectral 0.13.5

Thank you for reporting an issue !
